tests/acceptance/cli_test.py

```diff
@@ -264,6 +264,74 @@ def test_build_failed_forcerm(self):
         ]
         assert not containers
 
+    def test_create(self):
+        self.dispatch(['create'])
+        service = self.project.get_service('simple')
+        another = self.project.get_service('another')
+        self.assertEqual(len(service.containers()), 0)
+        self.assertEqual(len(another.containers()), 0)
+        self.assertEqual(len(service.containers(stopped=True)), 1)
+        self.assertEqual(len(another.containers(stopped=True)), 1)
+
+    def test_create_twice(self):
+        self.dispatch(['create'])
+        self.dispatch(['create'])
+        service = self.project.get_service('simple')
+        another = self.project.get_service('another')
+        self.assertEqual(len(service.containers()), 0)
+        self.assertEqual(len(another.containers()), 0)
+        self.assertEqual(len(service.containers(stopped=True)), 1)
+        self.assertEqual(len(another.containers(stopped=True)), 1)
+
+    def test_create_with_links(self):
+        self.base_dir = 'tests/fixtures/links-composefile'
+        self.dispatch(['create', 'web'], None)
+        web = self.project.get_service('web')
+        db = self.project.get_service('db')
+        console = self.project.get_service('console')
+        self.assertEqual(len(web.containers()), 0)
+        self.assertEqual(len(db.containers()), 0)
+        self.assertEqual(len(web.containers(stopped=True)), 1)
+        self.assertEqual(len(db.containers(stopped=True)), 1)
+        self.assertEqual(len(console.containers(stopped=True)), 0)
```

Comment:
Following the [testing pyramid](http://martinfowler.com/bliki/TestPyramid.html) I think the above two tests (create_twice and with_link) should be integration tests in `tests/integration/`. We already have plenty of end-to-end coverage with the other tests you've added.